wordpress-mobile/gutenberg-mobile
# Fix toolbar web breakage

Fix breakage from web

GB PR: https://github.com/WordPress/gutenberg/pull/22232

To test:
 - Make sure the demo app run and builds
 - Smoke test the demo app.

PR submission checklist:
